bootstrap.d/gui-apps.yml

```diff
@@ -0,0 +1,33 @@
+packages:
+  - name: swaybg
+    source:
+      subdir: ports
+      git: 'https://github.com/swaywm/swaybg'
+      branch: 'master'
+      commit: '3cb81c3573036a3606a4c7a1cedbe3e814a22773'
+      version: "1.0"
+    tools_required:
+      - system-gcc
+      - wayland-scanner
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: x86_64-managarm
+    pkgs_required:
+      - cairo
+      - wayland
+      - wayland-protocols
+    configure:
+      - args:
+        - 'meson'
+        - '--cross-file'
+        - '@SOURCE_ROOT@/scripts/meson-@OPTION:arch-triple@.cross-file'
+        - '--prefix=/usr'
+        - '--buildtype=release'
+        - '-Dman-pages=disabled'
+        - '-Dgdk-pixbuf=disabled'
```

Comment:
Why is gdk-pixbuf disabled? We have that ported so we can use it.

Reply:
post-install hooks are not quite working, so that's off the table right now.

Reply:
These have been fixed now, can we enable this?